### PR TITLE
test: Early error private methods delete (#1343)

### PR DESCRIPTION
**Pull request by jbhoosreddy**
Tracking Issue: #1343
#### Summary of changes:
1. Add delete error tests for different forms of private methods.

### CODE REVIEW

**Comment by jbhoosreddy** [comment]
@leobalter @rwaldron for review and suggestions.

Also, note for future PRs. If there is a change which is to be made in many other files, it's probably fine to comment it in one place, I'll make sure I update all files accordingly.

**Comment by jbhoosreddy** [comment]
@mkubilay @littledan also

**Comment by jbhoosreddy** [comment]
Also, is there a public slack for asking questions regarding test262 or related stuff? or IRC.

**Comment by leobalter** [comment]
I need a bit of dedicated time to read this one, but I'll come back with a proper review.

Meanwhile, @rwaldron would you mind to get your eyes in here?

**Comment by rwaldron** [comment]
> Also, is there a public slack for asking questions regarding test262 or related stuff? or IRC.

irc.mozilla.org: #test262

File: src/class-elements/err-delete-call-expression-private-method-async-gen.case [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+//- infieldsetup
+g = this.f;
+//- infunctionsetup
+var g = this.f;
+//- expression
+g().#m
+//- functiondeclaration
+  f() {
+  	return this;
```
[the remark this answers is not in the document]
Comment (jbhoosreddy) [reply src/class-elements/err-delete-call-expression-private-method-async-gen.case]: I don't mind making any changes at all. Thanks!

**Comment by jbhoosreddy** [comment]
@rwaldron for review!

**Comment by leobalter** [comment]
the contents of this PR were merged at #1761

**Pull request closed without merging**